eos/form-factors/mesonic.cc

```diff
@@ -578,4 +580,71 @@ namespace eos
 
         return result;
     }
+
+    /* Vacuum -> P P Processes */
+
+    FormFactors<VacuumToPP>::~FormFactors()
+    {
+    }
+
+    double
+    FormFactors<VacuumToPP>::abs2_f_p(const double & q2) const
+    {
+        return std::norm(this->f_p(q2));
+    }
+
+    double
+    FormFactors<VacuumToPP>::arg_f_p(const double & q2) const
+    {
+        const double arg = std::arg(this->f_p(q2));
+        if (arg < 0.0) {
+            return arg + 2.0 * M_PI;
```

Comment:
I would only do this on the python side I think

Reply:
I had explicitly asked Matthew to do it on the C++ side. I think here it makes a lot of sense to provide a consistent output. It also makes it possible to plots the phase in the ongoing analysis straight from a YAML file rather than using a Python script.